docker/containers/mysql-test/my.cnf

```diff
@@ -0,0 +1,3 @@
+[mysqld]
+collation-server = utf8mb3_czech_ci
```

Comment:
@marekdedic tedka posunul na mb4 viz https://github.com/skaut/Skautske-hospodareni/pull/2422/files#diff-f1b643bc272ac2a2eb5ac839704179f920dba5f87fe69f47190471b6273e7083R64

Reply:
Zachovával jsem to, co bylo v původním setupu. Není problém změnit.

Reply:
jasny, pravě proto píšu, že už je to v master jinak